### ISSUE TITLE
[Bug]: some function with parameters [DECIMAL128] will not implement but throw error 

### ISSUE THREAD

**Issue by ouyuanning** [matrixorigin/matrixone#3807]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

[SQL STATEMENT]: select BIT_AND(99999999999999999.99999), BIT_OR(99999999999999999.99999), BIT_XOR(99999999999999999.99999);
[EXPECT RESULT]:
BIT_AND(99999999999999999.99999)	BIT_OR(99999999999999999.99999)	BIT_XOR(99999999999999999.99999)
100000000000000000	100000000000000000	100000000000000000
[ACTUAL RESULT]:
Function 'bit_and' with parameters [DECIMAL128] will be implemented in future version.

**Comment by ouyuanning** [comment]
we fix #3785 then we get #3807

**Comment by ouyuanning** [comment]
fixed in 3808
bit function will throw error
variance  and stddev_pop will use float64
